org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/impl/XtextBuilder.java

```diff
@@ -537,6 +553,33 @@ protected void addInfosFromTaskAndClean(ToBeBuilt toBeBuilt, Task task, IProgres
 		}
 	}
 	
+	/**
+	 * @since 2.19
+	 */
+	protected void pollQueuedBuildData() {
+		boolean needRebuild = false;
+		if (pollQueuedBuildData(getProject())) {
+			needRebuild = true;
+		}
+		for(IProject project: interestingProjects) {
```

Comment:
interestingProjects is private. so maybe we should make it protected too or add a (final) getter

Reply:
👍